pipeline/transformer/transformer_sierra/src/main/scala/weco/pipeline/transformer/sierra/transformers/SierraProduction.scala

```diff
@@ -17,5 +17,5 @@ object SierraProduction
     bibId: SierraBibNumber,
     bibData: SierraBibData
   ): List[ProductionEvent[IdState.Unminted]] =
-    MarcProduction(bibData)
+    MarcProduction(bibData, prefer264Field = true)
```

Comment:
I might have inverted this because EBSCO is probably the odd case, but not blocking for merge.